### ISSUE TITLE
Service passes traffic to invalid port when running as ClusterIP

### ISSUE THREAD

**Issue by rjhenry** [YOURLS/charts#56]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps) guide.

### Version

1.8.2

### Description

A Helm release install with ingress enabled and TLS true creates a Service which passes traffic to yourls-pod-name:443, while the pod itself is only listening on :80. This results in the ingress controller noting that the traffic is invalid.

### Expectation

Traffic be passed to a port on which the pod(s) is listening.

### Reproduction steps

Deploy a Helm release with
```yaml
ingress:
  enabled: true
  <snip>
  hosts:
    - name: example-site.tld
      path: /
      tls: true
      tlsSecret: example-site-tld-tls
```

### Context
A current workaround is to deploy - in parallel - a separate Service and Ingress object with the same selector, but only passing traffic to `pod:80`.

**Comment by rjhenry** [comment]
On further inspection, I believe this may have been a red herring; it appears that the ingress is created and passes traffic to the service's TCP/80 port, and my issue was with the ingress not being created correctly with my ingress controller.

Closing this issue.
